src/components/pages/RoleSignup/Applications/Styles/menteeApplication.css

```diff
@@ -29,4 +29,19 @@
     .menteeApplication {
       padding: 0;
     }
-  }
\ No newline at end of file
+  }
+.ant-radio {
+  background-color: #ffff00!important;
```

Comment:
https://css-tricks.com/when-using-important-is-the-right-choice/

I'm not going to make you change this now. But know that when you go down a path of using `!important` property that tends to be a path of no return. It will override EVERYTHING whether you intend it to or not. 

Your app has been built with `.less` so that you can do themeing or overriding elements in a specific way. Again I'm going going to have you change this now but I wanted to call attention to it in the event you have some time to clean things up along the way.